src/operation.ts

```diff
@@ -439,15 +433,22 @@ export default class Operation {
    * Convert the operation into an array of JSON Schema schemas for each available type of parameter available on the
    * operation.
    *
-   * @param globalDefaults Contains an object of user defined schema defaults.
-   */
-  getParametersAsJsonSchema(globalDefaults?: Record<string, unknown>) {
-    if (this.parameterJsonSchema) {
-      return this.parameterJsonSchema;
-    }
-
-    this.parameterJsonSchema = getParametersAsJsonSchema(this, this.api, globalDefaults);
-    return this.parameterJsonSchema;
+   * @param opts
+   * @param opts.globalDefaults Contains an object of user defined schema defaults.
+   * @param opts.mergeIntoBodyAndMetadata If you want the output to be two objects: body (contains
```

Comment:
you're obviously working in this so maybe this is making more sense in my brain but... is "split" a better descriptor what's happening here than "merge"?

Reply:
It's definitely more "merge" than "split" because this is merging `query`, `path`, `cookie`, and `header` JSON Schema parameter objects into a single `metadata` JSON Schema object.